http/jsonapi/runtime/parameters_test.go

```diff
@@ -254,6 +255,62 @@ func TestScanNumericParametersInQueryFloatArrayFail(t *testing.T) {
 	}
 }
 
+func TestScanStringParameters(t *testing.T) {
+	req := httptest.NewRequest("GET", "/foo?value0=12%20asdd&value1=", nil)
```

Comment:
What happens if using `+` instead of `%20`? Maybe we should also define whether that will be a space or not.

Reply:
Yes, it does, generally, the URL escaping rules apply. I added it to the test to make that clear.